src/generate-type-defs.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+
+const distDir = __dirname + "/../dist/";
+
+const fs = require('fs');
+
+const specConsumer = require('shift-spec-consumer');
+const {nodes, enums, namedTypes} = specConsumer(fs.readFileSync(require.resolve('shift-spec-idl/spec.idl'), 'utf8'), fs.readFileSync(require.resolve('shift-spec-idl/attribute-order.conf'), 'utf8'));
```

Comment:
This is an incredibly long line. You can put arguments on their own lines to make it more readable.

Reply:
heh, I copied it right out of the [existing script](https://github.com/shapesecurity/shift-spec-js/blob/es2016/src/generate-spec.js#L24). will fix